vagrant-pxe-harvester/settings.yml

```diff
@@ -55,28 +55,28 @@ harvester_network_config:
       mac: 02:00:00:0D:62:E2
       cpu: 4
       memory: 16384
-      disk_size: 60G
+      disk_size: 150G
```

Comment:
Can we back out the disk_size changes? I purposely setup nodes with various disk sizes to test VM scheduling behaviors and disaster recovery scenarios.

Reply:
Nevermind, I see both Prometheus and Grafana volumes are there which takes up 50 and 10 GBs respectively.